app/assets/stylesheets/people.scss

```diff
@@ -45,3 +47,13 @@
     .btn-danger { margin-top: 9px; }
   }
 }
+
+#sharing_message {
+  &.entypo-check {
+    color: darken($brand-success, 20%);
```

Comment:
You missed the override for this for the dark theme.

https://github.com/diaspora/diaspora/blob/9fa032e4c91710df072291a26b5344429fd69d65/app/assets/stylesheets/color_themes/_color_theme_override_dark.scss#L58-L60

You moved this so it also selects the sharing state on the hovercard ... but you didn't change the override on the dark theme, which still only selects the profile, so the dark hovercard looks like this now:

![image](https://user-images.githubusercontent.com/458548/140669129-2a439bd9-694a-40d2-b917-8d3450b6f6f4.png)

Reply:
good work supertux88
(btw it saddens me to see "mara" as test subject here):
> ![image](https://user-images.githubusercontent.com/458548/140669129-2a439bd9-694a-40d2-b917-8d3450b6f6f4.png)

Reply:
Nice catch! We should have a checklist on the wiki about what to test while changing the UI: user not logged in, dark theme, mobile... Those are common things which tend to be forgotten.

Reply:
Fixed. This looks weird to me though, we should only need to set different variables to make the themes work. But this is another topic.